Documentation/ux_principles.md

```diff
@@ -0,0 +1,51 @@
+# Deluge UX Principles
+## What the Deluge Is
+
+The Deluge is a standalone instrument that also works as a controller/sequencer. It has two core features which set it apart from the competition:
+
+* Tracks can be smoothly changed from the internal synths to midi/cv and back again
+* It features a linear arranger 
+
+These features should be maintained and any new features must not impact them
+
+## What the Deluge Isn't
+The deluge is not a computer, and should not feel like you're operating one. Direct complexity and options should be minimized as much as possible, instead allowing the user to create complexity through combining existing features. An example of this is envelope shape - the envelope attack shape is fixed, but through modulating the attack rate with the envelope it can be smoothly morphed from logarithmic to exponential.
+
+## Guidelines
+
+* Favour explicitness 
+    - Parameters describe their action on the waveform, not their result on the final sound
+    - A user who understands synthesis should immediately be able to edit the deluge
+* Favour simplicity
+    - Menus should never be mandatory
+    - Make the shortcuts and gold knob system feel like a feature and not a hindrance 
+    - The OLED screen is a bonus, not a requirement
+* Favour performance
+    - Entire parameter ranges should be useful and scaled for quick editing
+    - Gold knob twists control frequently edited parameters
+    - Shortcuts control infrequently edited paramters
```

Comment:
I would add an item after this "Menu-only items are reserved for rarely edited parameters and should be used when no other reasonable implementation option exists"

Reply:
@trappar But that statement will become less and less true over time. As more high-impact features are added, with no shortcuts or reasonable button combos, they will have to live in menus, surely?

Reply:
This is deliberate - it indicates that within the community firmware there will be an emphasis on UX, and a feature that is cool but not a good UX may not get merged

I think it's ok that for some things the answer is to take advantage of the deluge's fantastic sequencer and integrate something else to your setup!